android/build.gradle

```diff
@@ -135,7 +135,7 @@ dependencies {
     //noinspection GradleDynamicVersion
   implementation "com.facebook.react:react-native:+"
   implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
-  implementation 'io.rownd:android:1.3.3'
+  implementation 'io.rownd:android:1.6.6'
```

Comment:
1.7.0 dropped last night, so can we bump up to that?

Reply:
Android and iOS have been updated